frontend/public/components/namespace.jsx

```diff
@@ -268,22 +268,18 @@ class NamespaceDropdown_ extends React.Component {
 
 const NamespaceDropdown = connect(namespaceDropdownStateToProps)(NamespaceDropdown_);
 
-const NamespaceSelector_ = ({flags}) => {
-  const openshiftFlag = flags[FLAGS.OPENSHIFT];
-  if (flagPending(openshiftFlag)) {
-    // Wait until the flag is initialized.
-    return <div className="co-namespace-selector" />;
-  }
-
-  const model = getModel(openshiftFlag);
-  const resources = [{ kind: model.kind, prop: 'namespace', isList: true }];
-
-  return <Firehose resources={resources}>
-    <NamespaceDropdown useProjects={openshiftFlag} />
+const NamespaceSelector_ = ({useProjects, inFlight}) => inFlight
```

Comment:
Use the models registered using API discovery to determine if `Projects` or `Namespaces` should be used.